pkg/testing/integration/program.go

```diff
@@ -1821,6 +1822,36 @@ func getVirtualenvBinPath(cwd, bin string) (string, error) {
 	return virtualenvBinPath, nil
 }
 
+// getSanitizedPkg strips the version string from a go dep
+func getSanitizedPkg(pkg string) string {
+	re := regexp.MustCompile(`v\d`)
+	v := re.FindString(pkg)
+	if v != "" {
+		return strings.TrimSuffix(strings.Replace(pkg, v, "", -1), "/")
+	}
+	return pkg
+
+}
+
+func getRewritePath(pkg string, gopath string, depRoot string) string {
+
+	var depParts []string
+	sanitizedPkg := getSanitizedPkg(pkg)
+
+	splitPkg := strings.Split(sanitizedPkg, "/")
+
+	if depRoot != "" {
+		// Get the package name
+		// This is the value after "github.com/foo/bar"
+		pkgName := splitPkg[2]
```

Comment:
pkg is kind of confusing name here as well. In terms of `github.com/org/repo` this is `repo` right?

Reply:
yeah, would `repoName` be better?

Reply:
changed the name here